### PR TITLE
Gerudo objects

### PR DESCRIPTION
**Pull request by EllipticEllipsis**
Decompiled `object_ge1`, `object_gla` and `object_nb` completely and add symbols to their respective actors. Ge1 is also used in `DemoEc`, but all of the actor symbols in `DemoEc` should be done in one pass imo. Added limb enums to EnNb since several are used. Also one rename in gameplay_keep per @Zelllll 's suggestion, and changed the limb enum in GeldB to use hex as recommended by @Roman971 .

Some of the animation names are a bit verbose, but I think it's mostly worth it for the sake of clarity, especially while the actor is not completely documented.

Will probably document EnNb properly at some point, but have more urgent things to do now.

### CODE REVIEW

File: src/overlays/actors/ovl_En_GeldB/z_en_geldb.h [hunk 1]
```diff
@@ -7,31 +7,31 @@
...
+    /* 0x0F */ GELDB_LIMB_L_HAND,
+    /* 0x10 */ GELDB_LIMB_L_SWORD,
+    /* 0x11 */ GELDB_LIMB_L_THIGH,
+    /* 0x12 */ GELDB_LIMB_L_SHIN,
+    /* 0x13 */ GELDB_LIMB_L_FOOT,
+    /* 0x14 */ GELDB_LIMB_R_THIGH,
+    /* 0x15 */ GELDB_LIMB_R_SHIN,
+    /* 0x16 */ GELDB_LIMB_R_FOOT,
+    /* 0x17 */ GELDB_LIMB_WAIST,
+    /* 0x18 */ GELDB_LIMB_MAX
```
Comment (louist103) [inline src/overlays/actors/ovl_En_GeldB/z_en_geldb.h:34]: Limbs should decimal IMO but if someone said to use hex leave it.

File: src/overlays/actors/ovl_En_Nb/z_en_nb.c [hunk 2]
```diff
@@ -797,7 +778,7 @@ void EnNb_CheckIfKneeling(EnNb* this, s32 animFinished) {
 }
 
 void EnNb_SetupLookRight(EnNb* this) {
-    AnimationHeader* animation = &D_06003954;
+    AnimationHeader* animation = &gNabooruOnAllFoursToOnOneKneeLookingRightTransitionAnim;
     f32 frames = Animation_GetLastFrame(animation);
```
Comment (louist103) [inline src/overlays/actors/ovl_En_Nb/z_en_nb.c:782]:
```suggestion
    f32 lastFrame= Animation_GetLastFrame(animation);
```

File: src/overlays/actors/ovl_En_Nb/z_en_nb.c [hunk 3]
```diff
@@ -817,7 +798,7 @@ void EnNb_CheckIfLookingRight(EnNb* this, s32 animFinished) {
 }
 
 void EnNb_SetupLookLeft(EnNb* this) {
-    AnimationHeader* animation = &D_06002DBC;
+    AnimationHeader* animation = &gNabooruOnOneKneeTurningHeadRightToLeftTransitionAnim;
     f32 frames = Animation_GetLastFrame(animation);
```
Comment (louist103) [inline src/overlays/actors/ovl_En_Nb/z_en_nb.c:802]:
```suggestion
    f32 lastFrame = Animation_GetLastFrame(animation);
```

File: src/overlays/actors/ovl_En_Nb/z_en_nb.c [hunk 4]
```diff
@@ -843,7 +824,7 @@ void EnNb_SetupDemo6KInConfrontation(EnNb* this, GlobalContext* globalCtx, s32 a
 }
 
 void EnNb_SetupRun(EnNb* this) {
-    AnimationHeader* animation = &D_06006320;
+    AnimationHeader* animation = &gNabooruKneeingToRunningToHitAnim;
     f32 frames = Animation_GetLastFrame(animation);
```
Comment (louist103) [inline src/overlays/actors/ovl_En_Nb/z_en_nb.c:828]:
```suggestion
    f32 lastFrame = Animation_GetLastFrame(animation);
```

File: src/overlays/actors/ovl_En_Nb/z_en_nb.c [hunk 5]
```diff
@@ -1294,10 +1275,10 @@ void func_80AB3A7C(EnNb* this, GlobalContext* globalCtx, s32 animFinished) {
 
     if ((u16)(kREG(17) + 0x19) > movementTimer) {
```
Comment (louist103) [inline src/overlays/actors/ovl_En_Nb/z_en_nb.c:1276]:
```suggestion
    if ((u16)(kREG(17) + 25) > movementTimer) {
```

**Review by louist103 — CHANGES_REQUESTED** [review]

File: src/overlays/actors/ovl_En_GeldB/z_en_geldb.h (hunk 1 above)
Comment (EllipticEllipsis) [reply src/overlays/actors/ovl_En_GeldB/z_en_geldb.h:34]: I asked in Discord and @Roman971 said all enums in hex if possible: https://discord.com/channels/688807550715560050/688851317593997489/828722876801613895

**Review by AngheloAlf — APPROVED** [review]

**Review by louist103 — APPROVED** [review]

**Review by fig02 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Pull request merged**